README.md

```diff
@@ -1,7 +1,7 @@
 # cribbage
-This repo is a little free-time exercise that has turned into a little free-time project.
+This repo started as a challenge from a friend to calculate the distribution of all hands' points for every possible cribbage hand faster than his Matlab implementation. Spoilers: golang is faster than Matlab. Now it's a place to teach some friends how to code and a playground to mess with unfamiliar technologies in golang (like [mongodb](https://www.mongodb.com), [web assembly](https://webassembly.org/), and hopefully [aws lambdas](https://aws.amazon.com/lambda/) and more).
```

Comment:
🌵 usually written as "MATLAB"

Reply:
did you ever publish that solution anywhere? or is it just local?